src/Cake.NuGet/Install/NuGetSourceRepositoryProvider.cs

```diff
@@ -105,28 +112,16 @@ SourceRepository GetOrCreateRepository(string source)
             }
         }
 
-        public IEnumerable<SourceRepository> GetPrimaryRepositories() => _primaryRepositories;
-
-        public IEnumerable<SourceRepository> GetRepositories() => _repositories;
-
-        public SourceRepository CreateRepository(string source) => CreateRepository(new PackageSource(source));
+        public IEnumerable<SourceRepository> PrimaryRepositories => _primaryRepositories;
 
-        public SourceRepository CreateRepository(PackageSource source) => CreateRepository(source, FeedType.Undefined);
+        public IEnumerable<SourceRepository> LocalRepositories => _localRepositories;
 
-        public SourceRepository CreateRepository(PackageSource source, FeedType type)
-        {
-            var repository = new SourceRepository(source, _resourceProviders, type);
-            var comparer = new SourceRepositoryComparer();
+        public IEnumerable<SourceRepository> Repositories => _repositories;
 
-            if (_repositories.Contains(repository))
-            {
-                return _repositories.First(x => comparer.Equals(x, repository));
-            }
+        private SourceRepository CreateRepository(string source) => CreateRepository(new PackageSource(source));
```

Comment:
Don't use expression-bodied methods. We're not using it anywhere else in the code base.